aspnetcore/fundamentals/http-requests.md

```diff
@@ -0,0 +1,241 @@
+---
+title: Initiate HTTP requests
+author: stevejgordon
+description: Learn about using the IHttpClientFactory interface to manage logical HttpClient instances in ASP.NET Core.
+manager: wpickett
+monikerRange: '>= aspnetcore-2.1'
+ms.author: scaddie
+ms.custom: mvc
+ms.date: 04/05/2018
+ms.prod: asp.net-core
+ms.technology: aspnet
+ms.topic: article
+uid: fundamentals/http-requests
+---
+# Initiate HTTP requests
+
+By [Glenn Condron](https://github.com/glennc), [Ryan Nowak](https://github.com/rynowak), and [Steve Gordon](https://github.com/stevejgordon)
+
+An `IHttpClientFactory` can be registered and used to configure and create [HttpClient](/dotnet/api/system.net.http.httpclient) instances in an app. It offers the following benefits:
+
+* Provides a central location for naming and configuring logical `HttpClient` instances. For example, a "github" client can be registered and configured to access GitHub. A default client can be registered for other purposes.
+* Codifies the concept of outgoing middleware via delegating handlers in `HttpClient` and provides extensions for Polly-based middleware to take advantage of that.
+* Manages the pooling and lifetime of underlying `HttpClientMessageHandler` instances to avoid common DNS problems that occur when manually managing `HttpClient` lifetimes.
+
```

Comment:
I'd suggest adding another bullet point here:

* Adds a configurable logging experience (via `ILogger`) for all requests sent through clients created by the factory.